app/controllers/items_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
       @merchant = Merchant.find(params[:merchant_id])
       @items = @merchant.items
     else
-      @items = Item.all
+      @items = Item.active_items
     end
```

Comment:
Love that we made a model method for this!

Reply:
WOO!!! I'm glad we are focusing on that MVC organization